documentation/how-to-branch.md

```diff
@@ -1,96 +1,87 @@
-# This will show you how you will branch from your respective work branch. 
+# Branching Guide for Development Teams
 
-Each team has their own main working branch `FE-Offers` `FE-Listing` etc. 
-From this main working branch each dev will make the branch for the task their working on. 
+## Overview
 
-ex: 
-Ola Nordmann is working on creating a listener for createOffer. Ola will then create a branch from `FE-Offers` that is named 
-```
-FE-Offers-createOfferListener
-```
-and then add the file for the listener in that branch. 
-This will give a more sense of ownership, easier for the general overview when you know who has been working on something,
-rather then spending time figure out who has worked on what files. 
+It's recommended for each development team to have its own main working branch (e.g., `teamName-Offers`, `teamName-Listing`). Developers are encouraged to create task-specific branches from their team's main working branch for their individual tasks. However, it's important to consult with the team leads or teachers before proceeding to ensure alignment with the overall project strategy.
```

Comment:
Is this the recommended branching strategy we're using now? Not the feature branch strategy?

Reply:
I just changed this to be more general, but after thinking about it I think I should perhaps change it to explain how github will give it the issue number if created from the issue..? @Sovak3441 @kyrregjerstad

Reply:
I'm thinking about this though: 

> `It's recommended for each development team to have its own main working branch (e.g., `teamName-Offers`, `teamName-Listing`)`

should we not encourage teams to use feature branches as we are doing now?


```suggestion
Our branching strategy utilizes the `main` branch as the central hub for all development. Developers are required to create new branches off `main` for every feature or bug fix, with the branch name starting with a keyword followed by a forward slash to indicate the type of change, such as feature/, fix/, etc. This is then followed by a description of the purpose, like feature/add-login or fix/header-layout. 

Once development on these branches is complete, a Pull Request (PR) should be initiated to merge the changes back into `main`, ensuring a thorough code review for maintaining quality standards. Branch names should be kept straightforward, avoiding special characters and relying on alphanumeric characters and hyphens for better clarity and compatibility. 
It's crucial for developers to align their work with the overall project strategy before commencing new features or fixes.
```

Reply:
yeah that looks better :)